_inc/client/appearance/index.jsx

```diff
@@ -37,16 +40,51 @@ export const Page = ( props ) => {
 		getModule
 	} = props,
 		isAdmin = props.userCanManageModules,
+		infiniteScrollDesc = getModule( 'infinite-scroll' ).description,
+		noInfiniteScrollSupport = ! props.themeData.support.infiniteScroll,
 		moduleList = Object.keys( props.moduleList );
 
+	if ( noInfiniteScrollSupport ) {
+		infiniteScrollDesc =
+			<div>
+				{ infiniteScrollDesc }
+				<p className="jp-form-setting-explanation">
+					{
+						__( "Your theme %(theme)s doesn't currently support Infinite Scroll, which needs data from your theme to function properly.",
+							{
+								args: {
+									theme: props.themeData.name
+								}
+							}
+						)
+					}
+				</p>
+				{
+					props.themeData.hasUpdate
+						? <p className="jp-form-setting-explanation">
```

Comment:
Yea, functionally it's the same. I just think it makes it easier to see the indentations. Just something that I've been doing personally, and thought I'd share.